### ISSUE TITLE
PR: Using inputbox warnings as showing errors is not accessible

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode-pull-request-github#2399]
https://github.com/microsoft/vscode/issues/114967

Try to create a PR with an empty title through the new create PR flow.
Notice the error under the input.

However turn on the screen reader and notice that nothing is being read out.

So if a user tries to create a PR with no title we should properly read out the error to the user.

**Comment by RMacfarlane** [comment]
Fixing two other things as part of this:
- The label for the "Create as draft" checkbox was not being read
- If the call to create fails, the error message is displayed at the bottom of the form, and also was not being read (to verify, turn off wifi before hitting create)

**Comment by connor4312** [comment]
NVDA did not read this pull request error

![](https://memes.peet.io/img/21-02-2d584dc7-ca96-4e08-a0c8-d862a5681b01.png)

I was curious why, aria-live is there and it _seems_ like it should work, but doesn't. However, adding `role="alert"` makes NVDA read it.

**Comment by RMacfarlane** [comment]
Thanks for catching! VoiceOver and NVDA seem to have slightly different behaviors for aria-live

**Comment by isidorn** [comment]
If you say `role="alert"` the screen reader will interrupt whatever is announcing and read it.
If you do not set anything it should announce it only once it has nothing left to announce
